client/js/components/statement/listStatements/ListStatements.vue

```diff
@@ -8,23 +8,35 @@
 </license>
 
 <template>
-  <div>
+  <div
+    :class="{ 'top-0 left-0 w-full h-full fixed z-[2200] bg-white': isFullscreen }">
     <!-- Header -->
     <dp-sticky-element
       border
-      class="py-2">
-      <div class="flex mb-2">
-        <search-modal
-          :search-in-fields="searchFields"
-          @search="(term, selectedFields) => applySearch(term, selectedFields)"
-          ref="searchModal" />
+      class="py-4"
+      :class="{ 'fixed top-0 left-0 w-full h-1/6 px-4': isFullscreen }">
+      <div class="flex items-center justify-between mb-2">
+        <div class="flex">
+          <search-modal
+            :search-in-fields="searchFields"
+            @search="(term, selectedFields) => applySearch(term, selectedFields)"
+            ref="searchModal" />
+          <dp-button
+            class="ml-2"
+            variant="outline"
+            data-cy="listStatements:searchReset"
+            :href="Routing.generate('dplan_procedure_statement_list', { procedureId: procedureId })"
+            :disabled="searchValue === ''"
+            :text="Translator.trans('search.reset')" />
+        </div>
         <dp-button
-          class="ml-auto"
+          data-cy="editorFullscreen"
+          :icon="isFullscreen ? 'compress' : 'expand'"
```

Comment:
One thing that must not be solved within this PR but keeps bothering me for some time now: we should look tweak padding of the button component in a way that the overall button height matches the height of text buttons when icon size is set to "medium".

Reply:
could it be resolved like that? --> https://github.com/demos-europe/demosplan-core/pull/2979/files#diff-231b07cc03a3b86b448c0aab75cb22ca2973a6acdcd57e5720282ce647702a07R268-L269 and https://github.com/demos-europe/demosplan-ui/pull/834/files#diff-2093ccfcda9bbc18eff049728bd6f2ab15a5864593932b4a9c57aa446a469001R161